backend/src/test/kotlin/cz/loono/backend/api/service/HealthcareProvidersServiceTest.kt

```diff
@@ -24,6 +24,7 @@ import kotlin.io.path.getLastModifiedTime
 
 @SpringBootTest(properties = ["spring.profiles.active=test"])
 @Transactional
+@Disabled
```

Comment:
Reason? :-) Pridej prosim nejaky komentar pripadne.

Reply:
Nema to zadny reason, to jsem vypnul jenom omylem :) Diky, [198e0c3](https://github.com/cesko-digital/loono-be/pull/83/commits/198e0c3ad0eab820d35938c82b60379d17c8341f)